### PR TITLE
Jesse we need to cook | You can now sell meth

### PR DESCRIPTION
**Pull request by Bibby0110**
# Document the changes in your pull request

(pseduo)Ephedrine can no longer be made but must be bought for 750cr per 20 units from cargo. 
Cooked meth can be thrown through a teleporter to sell it for 25 credits per unit,

~~lol darkrp~~

# Changelog

:cl:  
rscadd: Jesse we need to cook! Buy Sudafed through cargo and produce meth. Sell your meth through the teleporter. 
/:cl:

### CODE REVIEW

**Comment by Marmio64** [comment]
Is this just to hide a meth nerf that makes chemists need to buy from cargo to make it?

**Comment by Bibby0110** [comment]
> Is this just to hide a meth nerf that makes chemists need to buy from cargo to make it?

no this is a meme pr

**Comment by KillerOrcaCora** [comment]
JESSE!!!

**Comment by RG4ORDR** [comment]
We make our own methanlyn, bad PR you didn't even clean your lab.

File: code/modules/hydroponics/grown/cannabis.dm [hunk 1]
```diff
@@ -69,7 +69,6 @@
 						/datum/reagent/lithium = 0.15,
 						/datum/reagent/medicine/atropine = 0.15,
 						/datum/reagent/medicine/haloperidol = 0.15,
-						/datum/reagent/drug/methamphetamine = 0.15,
```
Comment (VaelophisNyx) [inline code/modules/hydroponics/grown/cannabis.dm:72]: uhhhhhh why?

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 2]
```diff
@@ -125,13 +125,6 @@
 	required_temp = 370
 	mix_message = "The mixture rapidly turns into a dense pink liquid."
 
-/datum/chemical_reaction/ephedrine
```
Comment (VaelophisNyx) [inline code/modules/reagents/chemistry/recipes/medicine.dm:128]: but why the ephedrine removal???

File: code/modules/reagents/chemistry/recipes/drugs.dm [hunk 3]
```diff
@@ -24,8 +24,8 @@
 /datum/chemical_reaction/methamphetamine
 	name = /datum/reagent/drug/methamphetamine
 	id = /datum/reagent/drug/methamphetamine
-	results = list(/datum/reagent/drug/methamphetamine = 4)
```
Comment (VaelophisNyx) [inline code/modules/reagents/chemistry/recipes/drugs.dm:27]: Strange choice to nerf meth production, which already is dangerous as fuck

File: code/datums/diseases/advance/advance.dm [hunk 4]
```diff
@@ -57,7 +57,7 @@
 										/datum/reagent/medicine/leporazine, /datum/reagent/toxin/mindbreaker, /datum/reagent/medicine/corazone
 									),
 									list(	// level 8
-										/datum/reagent/pax, /datum/reagent/drug/happiness, /datum/reagent/medicine/ephedrine
```
Comment (VaelophisNyx) [inline code/datums/diseases/advance/advance.dm:60]: should leave it as a cure

File: code/game/machinery/teleporter.dm [hunk 5]
```diff
@@ -72,6 +74,15 @@
...
 		visible_message(span_alert("Cannot authenticate locked on coordinates. Please reinstate coordinate matrix."))
 		return
+	if (istype(M, /obj/item/reagent_containers))
+		var/obj/item/reagent_containers/container = M
+		if(container.reagents && container.reagents.has_reagent(/datum/reagent/drug/methamphetamine))
+			var/volume = container.reagents.get_reagent_amount(/datum/reagent/drug/methamphetamine)
+			if(volume)
+				new /obj/item/holochip(drop_location(com), METH_PRICE * volume)
+				qdel(M)
+				return
```
Comment (tattax) [inline code/game/machinery/teleporter.dm:84]:
```suggestion
	if (istype(M, /obj/item/reagent_containers))
		var/obj/item/reagent_containers/container = M
		if(container.reagents?.has_reagent(/datum/reagent/drug/methamphetamine))
			var/volume = container.reagents.get_reagent_amount(/datum/reagent/drug/methamphetamine)
			if(volume)
				new /obj/item/holochip(drop_location(com), METH_PRICE * volume)
				qdel(M)
				return
```

File: code/modules/reagents/chemistry/recipes/medicine.dm (hunk 2 above)
Comment (Bibby0110) [reply code/modules/reagents/chemistry/recipes/medicine.dm:128]: someone can’t read lol (it’s you)

**Comment by Hopekz** [comment]
> no this is a meme pr

The teleporter thing actually seems like a real feature that would fit here

**Pull request closed without merging**